src/sw.js

```diff
@@ -0,0 +1,46 @@
+const version = "v0.0.1";
```

Comment:
This needs to be bumped for each update to staging and production for changes in the .html or .js files to be realised. We'll need to discuss the best method for this.

Reply:
This is one way of doing it [d74a44f](https://github.com/microbit-foundation/micropython-microbit-v2-simulator/pull/115/commits/d74a44fe4ffe80285e10175183abcfd86b1377d9)